docs/robots/warthog/maintenance_warthog.mdx

```diff
@@ -14,35 +14,127 @@ import Support from "/components/support.mdx";
+     modules in the system, the contactor will not close, and the system will not power up.
+2. Similarly, if a user is in discharge mode (the Warthog is in use) and any errors are detected, or if the SOC falls to a cut off point,
+   the contactor will open and remove power from the system to protect the modules.
+   - It is not good practice to run the state-of-charge (SOC) to 0%. This will decrease the life of the battery module. In general, keeping the SOC to a
+     minimum of 30-50% can increase the lifetime of the batteries.
+   - As a note, the system SOC is identical to the module with the lowest SOC in the system, so if three modules are at 45% SOC, and
+     one module is at 25%, then the SOC of the system is 25%, and the BMS will act accordingly cutting off power to protect the module with
+     the lowest SOC. Cell balancing and proper charging techniques are important to maintaining good battery health.
+3. The Warthog can be charged when the unit is powered and running, and can also be charged when the unit is not powered (main disconnect switch is open).
+   - The charger has been configured and wired in a way that the BMS is actually controlling the charger and can turn it off, and on, depending
+     on the status of the charging cycle.
+   - Even though the unit can be charged while running, this process does not allow for proper cell balancing to occur. It will allow for bulk charge
+     to occur to regain most of the charge in the system, however, repeatedly charging in this manner will allow each module to become out of balance with
+     the other modules. If this process if followed long enough the SOC values for each module can vary by a substantial amount. The larger the variation,
+     the longer it will take to fully balance the system. Therefore it is recommended (depending on use) that at least once every two to three weeks that
+     **the system be completely charged while the main disconnect switch is in the off position.** Doing so will allow the BMS to go through an
+     equalization phase where the charger is being turned on and off by the BMS to balance the voltages across each cell layer, in each module across the system.
+     **This is a very important step in maintain battery health, and getting the maximum lifetime out of the battery system.**
+   - Refer to [BMS ROS Topics](#bms-ros-topics) to view the necessary topics through the ROS API.
+4. To allow for this level of control there are three contactors in the system:
+   - The main contactor for discharge of the system.
+   - The charge contactor to connect to the output of the charger.
+   - The charge control relay to turn the charger outputs on and off and control the cell balancing or equalization phase of the charge cycle.
+
+#### BMS ROS Topics {#bms-ros-topics}
+
+The table below shows topics that will be very useful in monitoring battery health, as outlined in the section above.
```

Comment:
Should mention that this is only available for Warthogs with lithium batteries.

Reply:
Added note